zeebe/docs/generalized_distribution.md

```diff
@@ -66,17 +66,22 @@ metadata and raw record value are removed from the state.
 The diagram shows the flow of commands and events. What it doesn't highlight is the record that
 belongs to these.
 
-The `CommandDistributionRecord` is relatively simple. It contains 4 properties which are required to
+The `CommandDistributionRecord` is relatively simple. It contains 6 properties which are required to
```

Comment:
```suggestion
The `CommandDistributionRecord` is relatively simple. It contains 5 properties which are required to
```

🤡 Even better just change it to `It contains the following properties:` or something